source/_integrations/fibaro.markdown

```diff
@@ -72,7 +72,7 @@ plugins:
   type: boolean
   default: false
 device_config:
-  description: Lists device specific parameter or behaviour overrides.
+  description: Lists device specific parameter or behavior overrides.
```

Comment:
Do we use American English? Because both forms of writing are correct

Reply:
> The language of the documentation should be American-English.

<https://developers.home-assistant.io/docs/en/documentation_standards.html#general-documentation>